spec/node-spec.js

```diff
@@ -221,14 +221,23 @@ describe('node feature', () => {
       })
 
       let output = ''
-      child.stderr.on('data', (data) => {
+      function cleanup () {
+        child.stderr.removeListener('data', errorDataListener)
+        child.stdout.removeListener('data', outDataHandler)
+      }
+      function errorDataListener (data) {
         output += data
-        if (output.trim().startsWith('Debugger listening on ws://')) done()
-      })
-
-      child.stdout.on('data', (data) => {
+        if (output.trim().startsWith('Debugger listening on ws://')) {
+          cleanup()
+          done()
+        }
+      }
+      function outDataHandler (data) {
+        cleanup()
         done(new Error(`Unexpected output: ${data.toString()}`))
-      })
+      }
+      child.stderr.on('data', errorDataListener)
+      child.stdout.on('data', outDataHandler)
```

Comment:
@MarshallOfSound We probably could have used `stdout.once(...)` here to avoid removing the listener manually.

Reply:
@alexeykuzmin We only want an event from one of the streams (stdout or stderr), `once` would still allow each event listener to be called once resulting in `done` being called twice